subworkflows/local/utils_nfcore_skierfe_pipeline/main.nf

```diff
@@ -20,6 +20,81 @@ include { imNotification            } from '../../nf-core/utils_nfcore_pipeline'
 include { UTILS_NFCORE_PIPELINE     } from '../../nf-core/utils_nfcore_pipeline'
 include { workflowCitation          } from '../../nf-core/utils_nfcore_pipeline'
 
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    DEFINE DEPENDENCIES (FILES AND WORKFLOWS) FOR OTHER WORKFLOWS
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*/
+
+// nf-validation does not support contitional file and params validation,
+// add these here.
+//
+// For workflows:
+// - The cnv_calling workflow depends on the mapping and short_variant_calling workflows.
+//   It can't be run if either of --skip_mapping_wf or --skip_short_variant_calling is active.
+//
+// For files:
+// - Since the assembly workflow relies on the dipcall dipcall_par file,
+//   dipcall_par has to be set whenever --skip_assembly_wf is not active.
+//
+// For presets:
+// - Preset "pacbio" can't be run without --skip_methylation_wf.
+//
+
+def parameterDependencies = [
+    "workflow": [
+        "skip_assembly_wf"          : [],
+        "skip_short_variant_calling": ["skip_mapping_wf"],
+        "skip_snv_annotation"       : ["skip_mapping_wf", "skip_short_variant_calling"],
+        "skip_cnv_calling"          : ["skip_mapping_wf", "skip_short_variant_calling"],
+        "skip_phasing_wf"           : ["skip_mapping_wf", "skip_short_variant_calling"],
+        "skip_repeat_wf"            : ["skip_mapping_wf", "skip_short_variant_calling", "skip_phasing_wf"],
+        "skip_methylation_wf"       : ["skip_mapping_wf", "skip_short_variant_calling", "skip_phasing_wf"],
+    ],
+    "files": [
+        "dipcall_par"    : ["skip_assembly_wf"],
```

Comment:
Is there a reason for writing map's keys within double quotes here, and not in the parameterStatus map?